acceptance/bundle/init/default-python/output/my_default_python/.gitignore

```diff
@@ -0,0 +1,8 @@
+.databricks/
```

Comment:
> acceptance/bundle/init/default-python/output/my_default_python/.gitignore

Can we avoid too many nested directories? output/my_default_python can be merged into out-my-default-python.